theme_kit/models/theme.py

```diff
@@ -22,9 +24,19 @@ def _compute_code(self):
                 code = code + r.left_panel_id.less
             if r.content_id:
                 code = code + r.content_id.less
+            if r.custom_css:
+                code = code + r.custom_css
             if code:
-                compiled = self.generate_less2css(code)
-                r.code = compiled
+                code = self.generate_less2css(code)
+            if r.custom_js:
+                js_code = r.custom_js
+                js_code = 'try {' + js_code + '''
+                    } catch (err) {
+                      console.log('Error' + err.name + ":" + err.message + ". " + err.stack);
+                      alert('Error' + err.name + ":" + err.message + ". " + err.stack);
+                    }'''
+                code = code + '''<script type="text/javascript" id="custom_js">''' + js_code + '''</script>'''
+                r.code = code
             else:
                 r.code = code
```

Comment:
r.code = code
            else:
                r.code = code

can be simplified:

            r.code = code